SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -329,7 +329,15 @@ void USpatialPlayerSpawner::FindPlayerStartAndProcessPlayerSpawn(Schema_Object*
 	// forwarded worker knows to search for a PlayerStart.
 	if (PlayerStartActor == nullptr)
 	{
-		VirtualWorkerToForwardTo = NetDriver->LoadBalanceStrategy->WhoShouldHaveAuthority(*UGameplayStatics::GetGameMode(GetWorld()));
+		AGameModeBase* GameMode = UGameplayStatics::GetGameMode(GetWorld());
+		if (GameMode == nullptr)
+		{
+			UE_LOG(LogSpatialPlayerSpawner, Error, TEXT("Gamemode not found. Client worker ID: %lld."), ClientWorkerId);
+			PassSpawnRequestToNetDriver(SpawnPlayerRequest, PlayerStartActor);
```

Comment:
If this is an error state why are we doing more processing of SpawnPlayerRequest and PlayerStartActor?

Reply:
This has been removed.